src/core_plugins/kbn_vislib_vis_types/public/gauge.js

```diff
@@ -21,7 +21,7 @@ export default function GaugeVisType(Private) {
         type: 'gauge',
         addTooltip: true,
         addLegend: true,
-
+        isDisplayWarning: true,
```

Comment:
I'd turn this off by default. I understand this is a very slight incompatible change, but I think it's worth it.